libraries/AP_NavEKF2/AP_NavEKF2_PosVelFusion.cpp

```diff
@@ -1008,15 +1010,21 @@ void NavEKF2_core::selectHeightForFusion()
         activeHgtSource = HGT_SOURCE_BARO;
     } else if ((frontend->_altSource == 2) && ((imuSampleTime_ms - lastTimeGpsReceived_ms) < 500) && validOrigin && gpsAccuracyGood) {
         activeHgtSource = HGT_SOURCE_GPS;
+#if AP_BEACON_ENABLED
```

Comment:
make it use HGT_SOURCE_BCN instead of 3?

Reply:
test with hgt source=3 without beacon compiled in, make sure it doesn't blowup

Reply:
No longer playing around in EK2.